extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -1405,7 +1404,7 @@ export const BlockaidAssetScanLabel = ({
 
   return (
     <BlockaidWarningModal
-      header={`This asset was flagged as ${blockaidData.result_type}`}
+      header={`This asset was flagged as ${blockaidData.result_type.toLowerCase()}`}
```

Comment:
slightly unrelated - but I just noticed that occasionally we use uppercase and lower case when denoting something as malicious, so making sure it's uniform throughout